scipy/sparse/csgraph/tests/test_pydata_sparse.py

```diff
@@ -147,3 +147,37 @@ def test_min_weight_full_bipartite_matching(graphs):
     desired = func(sp.csc_matrix(A_dense)[0:2, 1:3])
 
     assert_equal(actual, desired)
+
+
+@pytest.mark.parametrize(
+    "func",
+    [
+        spgraph.shortest_path,
+        spgraph.dijkstra,
+        spgraph.floyd_warshall,
+        spgraph.bellman_ford,
+        spgraph.johnson,
+        spgraph.minimum_spanning_tree,
+    ]
+)
+@pytest.mark.parametrize(
+    "fill_value, comp_func",
+    [(np.inf, np.isinf), (np.nan, np.isnan)],
```

Comment:
Minor nit -- we should use `np.isposinf` or something. `np.isinf` allows both positive and negative infinities

Reply:
```suggestion
    [(np.inf, np.isposinf), (np.nan, np.isnan)],
```